teamscale_precommit_client/precommit_client.py

```diff
@@ -94,6 +89,23 @@ def _format_findings(self, findings):
                                              self.teamscale_client.get_finding_url(finding))
                     for finding in sorted_findings]
 
+    def _print_findings(self, caption, findings, empty_line=False):
+        has_findings = self.log_to_stderr and len(findings) > 0
```

Comment:
I don't understand this variable. It's supposed to be true if there are findings, but it also checks if we print to stderr? Please either fix the condition or change the variable name, whichever needs adjustment.